src/hub.js

```diff
@@ -1353,7 +1314,7 @@ document.addEventListener("DOMContentLoaded", async () => {
       scene.addEventListener("adapter-ready", async ({ detail: adapter }) => {
         // HUGE HACK Safari does not like it if the first peer seen does not immediately
         // send audio over its media stream. Otherwise, the stream doesn't work and stays
-        // silent. (Though subsequent peers work fine.) This only affects naf janus adapter
+        // silent. (Though subsequent peers work fine.) This only affects naf dialog adapter
```

Comment:
You removed the hack so should also remove this comment (and presumably the usage of `track`, `oscillator`, and `stream`)